Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1213,12 +1248,21 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
     }
 }
 
+auto UnifiedPDFPlugin::toContextMenuItemTag(int tagValue) const -> ContextMenuItemTag
+{
+    static constexpr std::array regularContextMenuItemTags { ContextMenuItemTag::OpenWithPreview, ContextMenuItemTag::SinglePage, ContextMenuItemTag::SinglePageContinuous, ContextMenuItemTag::TwoPages, ContextMenuItemTag::TwoPagesContinuous, ContextMenuItemTag::ZoomIn, ContextMenuItemTag::ZoomOut, ContextMenuItemTag::ActualSize };
```

Comment:
I would unwrap this to have one item per line.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/9c3bdc6957f556075aba0f8d8ec751bd3c4f96fc